concourse/content-event-resource/README.md

```diff
@@ -33,3 +33,41 @@ Install dependencies:
 ### Release the latest image to dockerhub
 
 `make release-latest`
+
+
+## Concourse Interface
+
+This Concourse Resource is used for checking and acquiring events from the event service application.
+
+## Source Configuration
+
+* `api_root`: 
```

Comment:
Should we add an example like `http://content-events-api:5000`? and `queued` for `status`.

Reply:
Yea something like this would be good.

```yaml
resource_types:
- name: content-event-resource
  type: docker-image
  source:
    repository: openstax/content-event-resource
    version: 0.1.0

resources:
- name: content-events-queued
  type: history
  source:
    api_root: http://content-events-api:5000
    status: queued
```

Reply:
oops, I totally missed putting in a description for these. It wasn't really the focus of what I'm working on, so I got a bit sloppy in trying to just get it done for what I needed.